sound/soc/codecs/rt5682-sdw.c

```diff
@@ -770,6 +773,7 @@ static int __maybe_unused rt5682_dev_resume(struct device *dev)
 
 regmap_sync:
 	slave->unattach_request = 0;
+	regcache_cache_only(rt5682->sdw_regmap, false);
```

Comment:
should we do a regcache_sync(rt5682->sdw_regmap); as well?

Reply:
@bardliao if we treat both regmaps at the same time, then we're missing a regcache_sync(rt5682->sdw_regmap) below, no?

Reply:
> @bardliao if we treat both regmaps at the same time, then we're missing a regcache_sync(rt5682->sdw_regmap) below, no?

@plbossart No, we don't do regcache_sync(rt5682->sdw_regmap);. The cache type of sdw_regmap is REGCACHE_NONE.